### ISSUE TITLE
Unable to enter edit mode with non-default layout

### ISSUE THREAD

**Issue by jackwilsdon** [thomasloven/lovelace-layout-card#153]
My Home Assistant version: 2021.7.1

Layout-card version (FROM BROWSER CONSOLE): 2.3.0

What I am doing: editing a dashboard with non-default layout

What I expected to happen: the dashboard would enter edit mode properly

What happened instead: the dashboard does not properly enter edit mode:

 ![screenshot showing issue](https://user-images.githubusercontent.com/1843197/125211926-dec50d00-e2a1-11eb-826c-ee6c02b2c170.png)

* The action bar does not appear below cards, meaning they cannot be edited directly or reordered.
* The add card button styling seems to be broken.

**Minimal** steps to reproduce:
1. Create a dashboard with a non-default layout and at least one card:
    ```yaml
    views:
      - type: custom:masonry-layout
        cards:
          - type: button
    ```
2. Reload the page (this is important - the issue does not appear until after a reload)

---

**By putting an X in the boxes ([X]) below, I indicate that I:**

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [x] Have made sure I am using the latest version of the plugin.
- [x] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.
- [x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by magenbrot** [comment]
same here. As a workaround: the edit button appears when entering edit-mode on a different dashboard (with default layout) and then switching back to the dashboard with the custom-layout.

**Comment by disforw** [comment]
Same issue here, I removed this plug-in and issue went away.

**Comment by CoryAlbrecht** [comment]
Me as well. Change layout and the problem goes away and the edit button returns. but I really want that different layout as the default is too limiting.

**Comment by Protoncek** [comment]
An update for this problem: 
it seems that this problem exists only if lovelace layout is set to anything but default. I have one lovelace borad set to "vertical" (for my mobile phone) i have this problem. If i set layout back to default all is ok and "edit" bars appear immediately.
So, going to ANY lovelace with default layout causes all other pages to go back to normal. But only until browser is closed and re-opened...

**Comment by sesquipedality** [comment]
Also observing this issue.  Have been working around by changing the layout to default then back to horizontal, but switching to a default tab sounds easier.

**Comment by Protoncek** [comment]
Today i experimented a bit again, and i can say that by removing "layout-card" problem completely dissapears. So, it's definitely this card's problem. So, i guess it'll have to be removed until this issue is resolved...

@thomasloven : do you have any clue what could be the problem?

**Comment by thomasloven** [comment]
This should be fixed in release 2.3.1

**Comment by magenbrot** [comment]
I can confirm that it's working again. Thanks for the fix, @thomasloven!
